tests/test_release_notes_generator.py

```diff
@@ -171,3 +172,55 @@ def test_generate_release_notes_latest_release_found_by_published_at(
     assert "- #122 _I1+bug_" in release_notes
     assert "- PR: #101 _Fixed bug_" not in release_notes
     assert "- PR: #102 _Fixed bug_" in release_notes
+
+
+# get_latest_release tests
```

Comment:
On line 79 there is extra print() command.

Reply:
Fixed in commit 3bd9b0f89377c110b90a3c003373aa1e44cc7930.